geomesa-cassandra/geomesa-cassandra-tools/src/main/scala/org/locationtech/geomesa/cassandra/tools/commands/CassandraDeleteFeaturesCommand.scala

```diff
@@ -0,0 +1,24 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
```

Comment:
do you care about copyright attribution? we can add a new user or organization if so.